dynawo/sources/Models/Modelica/Dynawo/Examples/Wind/WECC/WTG4ACurrentSource.mo

```diff
@@ -84,7 +84,7 @@ model WTG4ACurrentSource "WECC Wind Type 4A Model (including the plant controlle
     QMinPu = -0.4,
     RPu = 0,
     RateFlag = false,
-    RefFlag = true,
+    RefFlag = 1,
```

Comment:
You will have to apply this change to both WTG4ACurrentSource1.mo and WTG4ACurrentSource2.mo during the rebase operation.